content/05-expression.md

```diff
@@ -492,6 +492,43 @@ This syntax is also used to access types within packages in the form of `pack.Ty
 
 The typer ensures that an accessed field actually exist and may apply transformations depending on the nature of the field. If a field access is ambiguous, understanding the [resolution order](type-system-resolution-order) may help.
 
+##### since Haxe 4.3.0
+
+Null-safe field access is expressed using the `?.` operator. Such access checks if the object is not `null` first, and only then accesses the given field. If the object *is* `null`, then the entire expression has the value `null`.
+
+```haxe
+object?.fieldName
+```
+
+This is the same as:
+
+```haxe
+(object != null ? object.fieldName : null)
```

Comment:
I stumbled across this in my talk too because the general case is that the left side gets bound to a temporary variable:

```haxe
var tmp = object;
tmp != null ? tmp.fieldName : null
```

This is to deal with the situation when the `object` expression has side-effects. I definitely like the simple example, but saying that it is the same is not 100% accurate at expression-level.